client/go/internal/cli/cmd/waiter.go

```diff
@@ -33,6 +33,10 @@ func (w *Waiter) DeployService(target vespa.Target) (*vespa.Service, error) {
 
 // Service returns the service identified by cluster ID, available on target.
 func (w *Waiter) Service(target vespa.Target, cluster string) (*vespa.Service, error) {
+	return w.ServiceWithAuthMethod(target, cluster, "mtls")
+}
+
+func (w *Waiter) ServiceWithAuthMethod(target vespa.Target, cluster string, authMethod string) (*vespa.Service, error) {
```

Comment:
Or perhaps the debug message about which Authentication method is used could be set directly in there and when `VESPA_CLI_DATA_PLANE_TOKEN` we simply mention that it's found.

Reply:
Good idea, I'll look into the cli tracing system